.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```

Comment:
Typically, the whole `.idea/` directory is ignored. So what I suggest is to create `.gitignore` in the repository root, and just add `.idea/` there.